verilog/analysis/verilog_project.cc

```diff
@@ -156,6 +156,23 @@ absl::StatusOr<VerilogSourceFile*> VerilogProject::OpenFile(
   return &file;
 }
 
+bool VerilogProject::RemoveRegisteredFile(
+    absl::string_view referenced_filename) {
+  if (files_.erase(std::string(referenced_filename)) == 1) {
```

Comment:
can we change `VerilogProject::file_set_type` to some absl type ? They have heterogeneous lookup, so we can do the lookup with the string_view.

Reply:
Done